src/Data/Tree/AVL.hs

```diff
@@ -130,12 +126,16 @@ module Data.Tree.AVL
 
       -- * Proof type, ways to construct and eliminate
     , Proof (..)
-    , fullRehash
     , prune
     , checkProof
 
-      -- * For debug
+      -- * Serialisation helpers
+    , beforeSerialise
```

Comment:
What usecases are assumed for beforeSerialise, afterDeserialise and mapTilt? There where discussion that AVL+ API try to hide that AVL+ is a Tree so I assume that Tilt is exposed for some important use case